src/config.js

```diff
@@ -37,11 +37,16 @@ module.exports = (store) => {
         } catch (err) {
           return callback(err)
         }
-        if (key !== undefined && !_has(config, key)) {
-          return callback(new Error('Key ' + key + ' does not exist in config'))
+
+        if (typeof key === 'undefined') {
+          return callback(null, config)
+        }
+
+        if (typeof key !== 'string') {
+          return callback(new Error('Key ' + key + ' must be a string.'))
         }
-        const value = key !== undefined ? _get(config, key) : config
-        callback(null, value)
+
+        callback(null, _get(config, key, null) || new Error('Key ' + key + ' does not exist in config'))
```

Comment:
The error needs to be passed in the first param of callback.

Reply:
OMG ofc